docs/_posts/2015-09-30-react-render-and-top-level-api.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "React.render and the Top Level React API"
+author: jimandsebastian
+---
+
+
+When you're in React's world you are just building components that fit into other components. Everything is a component. Unfortunately not everything around you is built using React. At the root of your tree you still have to write some plumbing code to connect the outer world into React.
+
+The primary API for rendering into the DOM looks like this:
+
+```js
+ReactDOM.render(reactElement, domContainerNode)
```

Comment:
Is there a reason to not be more descriptive here?  The longer variable name doesn't detract from anything, but it is slightly less mysterious.  We know what a "container" is, but reminding a relatively new user that it's a DOM node they can mount into seems mildly useful.  Don't feel super strongly either way, let me know if you still want it changed.